yml/dp02_obscore.yaml

```diff
@@ -32,6 +33,18 @@ tables:
     ivoa:unit:
     datatype: char
     votable:arraysize: 64
+  - name: obs_title
+    "@id": "ObsCore.obs_title"
+    description: Brief description of dataset in free format
+    nullable: true
+    ivoa:ucd: meta.title;obs
+    votable:utype: DataID.title
+    tap:std: 1
+    tap:principal: 1
+    tap:column_index: 225
+    ivoa:unit:
+    datatype: char
+    votable:arraysize: "*"
```

Comment:
The underlying SQL type used for the in-place mod was `VARCHAR(256)` -- do we want `votable:arraysize: 256` here to more closely represent that reality?

Reply:
No, I'm comfortable with the variable-length declaration.  I'm rethinking this more generally.